internal/usecases/reports/get_assets_report_test.go

```diff
@@ -0,0 +1,357 @@
+		QuoteHash:         "mockQuoteHash5",
+		RequiredLiquidity: entities.NewWei(5000),
+		State:             quote.PegoutStateWaitingForDeposit,
+	},
+}
+
+// nolint:funlen
+func TestGetAssetsReportUseCase_Run(t *testing.T) {
+	ctx := context.Background()
+
+	rskAddress := "rskAddress"
+
+	wallet := mocks.NewBitcoinWalletMock(t)
+	wallet.On("GetBalance").Return(entities.NewWei(100000), nil)
+
+	lpMock := &mocks.ProviderMock{}
+	lpMock.On("RskAddress").Return(rskAddress)
+
+	rsk := new(mocks.RootstockRpcServerMock)
+	rpc := blockchain.Rpc{Rsk: rsk}
+
+	rsk.On("GetBalance", ctx, rskAddress).Return(entities.NewWei(100000), nil)
+
+	lp := new(mocks.ProviderMock)
+	lp.On("AvailablePeginLiquidity", ctx).Return(entities.NewWei(67500), nil)
+	lp.On("AvailablePegoutLiquidity", ctx).Return(entities.NewWei(85000), nil)
+
+	peginRepository := mocks.NewPeginQuoteRepositoryMock(t)
+	peginRepository.On("GetRetainedQuoteByState", ctx, quote.PeginStateWaitingForDeposit, quote.PeginStateWaitingForDepositConfirmations).Return(retainedPeginQuotes, nil)
+	pegoutRepository := mocks.NewPegoutQuoteRepositoryMock(t)
+	pegoutRepository.On("GetRetainedQuoteByState", ctx, quote.PegoutStateWaitingForDeposit, quote.PegoutStateWaitingForDepositConfirmations).Return(retainedPegoutQuotes, nil)
+
+	useCase := reports.NewGetAssetsReportUseCase(wallet, rpc, lpMock, lp, lp, peginRepository, pegoutRepository)
+
+	result, err := useCase.Run(ctx)
+
+	require.NoError(t, err)
+	require.Equal(t, result.RbtcLiquidity, big.NewInt(67500))
+	require.Equal(t, result.BtcLiquidity, big.NewInt(85000))
+	require.Equal(t, result.BtcBalance, big.NewInt(100000))
+	require.Equal(t, result.RbtcBalance, big.NewInt(100000))
+	require.Equal(t, result.RbtcLocked, big.NewInt(17500))
+	require.Equal(t, result.BtcLocked, big.NewInt(15000))
+	lp.AssertExpectations(t)
+	peginRepository.AssertExpectations(t)
+	pegoutRepository.AssertExpectations(t)
+	rsk.AssertExpectations(t)
+	lpMock.AssertExpectations(t)
+	wallet.AssertExpectations(t)
+}
+
+func TestGetAssetsReportUseCase_Run_btcBalanceError(t *testing.T) {
+	ctx := context.Background()
+
+	wallet := mocks.NewBitcoinWalletMock(t)
+	wallet.On("GetBalance").Return(nil, assert.AnError)
+
+	lpMock := &mocks.ProviderMock{}
+
+	rsk := new(mocks.RootstockRpcServerMock)
+	rpc := blockchain.Rpc{Rsk: rsk}
+
+	lp := new(mocks.ProviderMock)
+
+	peginRepository := mocks.NewPeginQuoteRepositoryMock(t)
+	pegoutRepository := mocks.NewPegoutQuoteRepositoryMock(t)
+
+	useCase := reports.NewGetAssetsReportUseCase(wallet, rpc, lpMock, lp, lp, peginRepository, pegoutRepository)
+
+	result, err := useCase.Run(ctx)
+
+	require.Error(t, err)
+	require.Equal(t, result.RbtcLiquidity, big.NewInt(0))
+	require.Equal(t, result.BtcLiquidity, big.NewInt(0))
+	require.Equal(t, result.BtcBalance, big.NewInt(0))
+	require.Equal(t, result.RbtcBalance, big.NewInt(0))
+	require.Equal(t, result.RbtcLocked, big.NewInt(0))
+	require.Equal(t, result.BtcLocked, big.NewInt(0))
+	lp.AssertExpectations(t)
```

Comment:
you only mocked the execution of GetBalance on the btc wallet, instead of `AssertExpectations` you should use `AssertNotCalled` for the ones you didn't set up the mock because the function is supposed to return earlier, the same happens with the other error tests of this file